test/plugins/preformat/query-active.js

```diff
@@ -0,0 +1,22 @@
+/** @jsx h */
+
+export default editor => {
+    let block = editor.value.startBlock
+
+    let node = block.type === 'preformat' ? block : null
+    should.not.exist(node)
+
+    editor.moveToStartOfNode(editor.value.document.getNode('preformat'))
+    block = editor.value.startBlock
+    node = block.type === 'preformat' ? block : null
+    should.exist(node)
```

Comment:
If `block` can be `null` or `undefined`, then `block.type` will fail. If it can't then there's no reason to use this elaborate construction with `should.exist` over a simple `block.type.should.equal('preformat')`.